markwon/src/main/java/ru/noties/markwon/renderer/SpannableRenderer.java

```diff
@@ -13,6 +12,6 @@
     public CharSequence render(@NonNull SpannableConfiguration configuration, @NonNull Node node) {
```

Comment:
I suggest that we change this method's return value to `SpannableBuilder` instead of `CharSequence`